contracts/token/ERC721/base/ERC721BaseInternal.sol

```diff
@@ -64,13 +64,12 @@ abstract contract ERC721BaseInternal is IERC721BaseInternal {
 
     function _mint(address to, uint256 tokenId) internal virtual {
         if (to == address(0)) revert ERC721Base__MintToZeroAddress();
-
-        ERC721BaseStorage.Layout storage l = ERC721BaseStorage.layout();
-
-        if (l.exists(tokenId)) revert ERC721Base__TokenAlreadyMinted();
+        if (_exists(tokenId)) revert ERC721Base__TokenAlreadyMinted();
```

Comment:
Would such functions benefit from having a layout parameter?

This would remove the need of more than one call of `layout()` in a single function.

Not sure if gas is more important than clarity here.

Reply:
No, I'm avoiding passing `l` as a parameter as a rule.  We used in in the Premia contracts because at the time our architecture required contract size savings, but there's always a way around that when necessary.